src/splat.F

```diff
@@ -18,34 +18,27 @@
 C> if an equator point exists, its weight should be halved.
 C> This version invokes the ibm essl matrix solver.
 C>
-C> PROGRAM HISTORY LOG:
-C> -  96-02-20  IREDELL
-C> -  97-10-20  IREDELL  ADJUST PRECISION
-C> -  98-06-11  IREDELL  GENERALIZE PRECISION USING FORTRAN 90 INTRINSIC
-C> - 1998-12-03  IREDELL  GENERALIZE PRECISION FURTHER
-C> - 1998-12-03  IREDELL  USES AIX ESSL BLAS CALLS
-C> - 2009-12-27  DSTARK   updated to switch between ESSL calls on an AIX
-C>                      platform, and Numerical Recipies calls elsewise.
-C> - 2010-12-30  SLOVACEK update alignment so preprocessor does not cause
-C>                      compilation failure
-C> - 2012-09-01  E.Mirvis & M.Iredell merging & debugging linux errors 
-C>			of _d and _8 using generic LU factorization.   
-C> - 2012-11-05  E.Mirvis  generic FFTPACK and LU lapack were removed
+C> ### Program History Log
```

Comment:
Move the whole history log to the file-documentation block, instead of the subroutine documentation block. See for example https://github.com/NOAA-EMC/NCEPLIBS-sp/pull/75/files#diff-30e83b76e0c0698560f5383533fe51ed65a1519eb001f15b932f756d3cdaa824